todo_app/templates/index.html

```diff
@@ -22,33 +22,73 @@ <h2>TODO Items</h2>
       </div>
 
       <form id="update_all_form" method="post" action="{{ url_for('update_todo') }}"></form>
-      <form id="update_single_form" method="post" action="/todo/update_item"></form>
+      <form id="update_single_form" method="post" action="{{ url_for('update_single_item') }}"></form>
 
       <table>
         <tr>
-          <th>Status</th>
-          <th>TODO Item</th>
-          <th>Description</th>
-          <th>Due Date</th>
-          <th>Toggle Status</th>
+          <th>TO DO</th>
+          <th></th>
+          <th>DONE</th>
         </tr>
-        {% for item in todo %}
         <tr>
-          <td>
-            <input form="update_all_form" type="checkbox" id="{{ item.id }}" name="{{ item.id }}" 
-                value="{{ item.idList }}" {{'checked' if item.idList==completed_list_id else ''}}>
-            </input>
+          <td valign="top">
+            <table>
+              <tr>
+                <th>Status</th>
+                <th>TODO Item</th>
+                <th>Description</th>
+                <th>Due Date</th>
+                <th>Toggle Status</th>
+              </tr>
+              {% for item in todo_view_model.todo_items %}
+              <tr>
+                <td>
+                  <input form="update_all_form" type="checkbox" id="{{ item.id }}" name="{{ item.id }}" 
+                      value="{{ item.idList }}" {{'checked' if item.status=='Done' else ''}}>
```

Comment:
Minor, but now that we're filtering by status we can simplify this:
```suggestion
                      value="{{ item.idList }}">
```

And similarly below for the done items:
```jinja
value="{{ item.idList }}" 'checked'>
```